### DISCUSSION TITLE
Navbar + Sidebar on mobile = drawer

### DISCUSSION THREAD

**Discussion by Falci** [themesberg/flowbite-react#1064] [Q&A]
I'm trying to recreate [this example](https://flowbite.com/docs/components/sidebar/#sidebar-with-navbar) but in react.
I couldn't find the correct components from `flowbite-react` and the `data-drawer-target/toggle` seems to have no effect in a react app.

What's the proper way to make it?

**Comment by idevN** [comment]
Has anyone found a solution for this yet?
